eth/vm/computation.py

```diff
@@ -140,6 +143,7 @@ def __init__(self,
 
         self._memory = Memory()
         self._stack = Stack()
+        self._rstack = RStack()
```

Comment:
Oh, also think that spelling out the full attribute name is friendlier, like:
```suggestion
        self._return_stack = ReturnStack()
```